### ISSUE TITLE
MCS code does not support stereochemistry

### ISSUE THREAD

**Issue by greglandrum** [rdkit/rdkit#6]
Thread here: http://www.mail-archive.com/rdkit-discuss@lists.sourceforge.net/msg02934.html

```
In [2]: mol1 = Chem.MolFromSmiles("Fc1ccc(cc1)[C@@]3(OCc2cc(C#N)ccc23)CCCN(C)C") 
In [3]: mol2 = Chem.MolFromSmiles("Fc1ccc(cc1)[C@]3(OCc2cc(C#N)ccc23)CCCN(C)C")

In [4]: from rdkit.Chem import MCS

In [6]: MCS.FindMCS((mol1,mol2))
Out[6]: MCSResult(numAtoms=24, numBonds=26, smarts='[F]-[#6]:1:[#6]:[#6]:[#6](-[#6]-2(-[#6]-[#6]-[#6]-[#7](-[#6])-[#6])-[#8]-[#6]-[#6]:3:[#6]:[#6](:[#6]:[#6]:[#6]:3-2)-[#6]#[#7]):[#6]:[#6]:1', completed=1)
```

**Comment by proteneer** [comment]
Is this still being worked on?

**Comment by greglandrum** [comment]
It's not actively being worked on

**Comment by proteneer** [comment]
I wouldn't mind working on a PR if you could point me to the relevants parts of the code.

**Comment by greglandrum** [comment]
The MCS code is now in C++. If you're comfortable with that, the code is here:  https://github.com/rdkit/rdkit/tree/master/Code/GraphMol/FMCS

It may be possible using the existing chirality support in the substructure matcher, but it's almost certainly a non-trivial amount of work.

**Comment by greglandrum** [comment]
this was closed a long time ago by #466. Closing it
